app/src/main/java/com/eveningoutpost/dexdrip/GcmListenerSvc.java

```diff
@@ -56,6 +56,8 @@
 import java.util.List;
 import java.util.Map;
 
+import static com.eveningoutpost.dexdrip.xdrip.gs;
```

Comment:
I don't think this import is used is it?

Reply:
Thanks for catching this;  sorry about that; removed.